### PR TITLE
Removing decrypted client secret in sap configuration from code

### PR DESCRIPTION
**Pull request by MueezKhan246**
Description:
Removing decrypted client secret in sap configuration from code
this PR can be merged as it has no dependency.

JIRA:
https://2u-internal.atlassian.net/jira/software/c/projects/ENT/issues/ENT-8972

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -17,6 +17,10 @@ Unreleased
 ----------
 * nothing unreleased
 
+[4.23.20]
+----------
+* feat: removed char field decrypted_secret from code
```
Comment (sameenfatima78) [inline CHANGELOG.rst:22]: nit: removed char field decrypted_secret references

**Review by sameenfatima78 — APPROVED** [review]
one comment, looks good otherwise

File: integrated_channels/sap_success_factors/models.py [hunk 2]
```diff
@@ -123,18 +123,6 @@ class SAPSuccessFactorsEnterpriseCustomerConfiguration(EnterpriseCustomerPluginC
...
-        blank=True,
-        default='',
-        verbose_name="Encrypted Client Secret",
-        help_text=_(
-            "The encrypted OAuth client secret."
-            " It will be encrypted when stored in the database."
-        ),
-        null=True
-    )
-
```
Comment (hamzawaleed01) [inline integrated_channels/sap_success_factors/models.py:137]: it should have a migration also?

File: CHANGELOG.rst (hunk 1 above)
Comment (MueezKhan246) [reply CHANGELOG.rst]: @sameenfatima78 updated

**Review by zamanafzal — APPROVED** [review]

**Pull request merged**